Cargo.toml

```diff
@@ -16,19 +16,27 @@
 # under the License.
 
 [package]
-authors = ["Apache Software Foundation <dev@opendal.apache.org>"]
-categories = ["filesystem"]
-description = "OpenDAL: Access data freely, painlessly, and efficiently."
-edition = "2021"
-exclude = [".github/", "bindings/", "profiles/", "oli/", "tests/"]
-homepage = "https://opendal.apache.org/"
-keywords = ["storage", "fs", "s3", "azblob", "gcs"]
-license = "Apache-2.0"
 name = "opendal"
-repository = "https://github.com/apache/incubator-opendal"
 version = "0.30.3"
-# MSRV of OpenDAL. Please update this field while bump.
-rust-version = "1.60"
+
```

Comment:
I prefer to keep all `toml` file sorted by `taplo` instead of organized by hand.

Reply:
OK